graylog2-web-interface/src/views/components/aggregationbuilder/SeriesSelect.tsx

```diff
@@ -130,10 +130,6 @@ class SeriesSelect extends React.Component<Props, State> {
     this.setState({ options: suggester.defaults });
   };
 
-  _onClose = () => {
```

Comment:
Removing this is breaking the behavior which resets the state to selecting a function when closing the select.

Reply:
I removed this part because the `react-select` component does not have a prop `onClose`. But I understand the benefit and will reimplement this function and use the prop `onMenuClose` instead.